client/packages/requisitions/src/RequestRequisition/DetailView/ContentArea.tsx

```diff
@@ -32,6 +34,24 @@ const useHighlightPlaceholderRows = (
   }, [rows, setRowStyles]);
 };
 
+export const ItemCell = <T extends RequestLineFragment>({
+  rowData,
+}: CellProps<T>): React.ReactElement<CellProps<T>> => {
+  const itemName = rowData.item.name;
+  return (
+    <Tooltip title={String(itemName)} placement="bottom-start">
+      <div
+        style={{
+          overflow: 'hidden',
+          textOverflow: 'auto',
+        }}
+      >
+        {String(itemName)}
+      </div>
+    </Tooltip>
+  );
+};
+
```

Comment:
This feels like it could be a generic component, however the type being passed to the item cell is different for two different lists, so I've made it twice.

I wonder if anyone knows how I could make this accept either type / be generic?

Reply:
Further to this, where it is shared between two lists (Inventory/Stock, and Internal Order), I wasn't sure where to put it. 

So have arbitrarily placed it in the StockListComponent, and then imported into the column file of RequestRequisitions.